rasa/core/nlg/interpolator.py

```diff
@@ -24,7 +24,7 @@ def interpolate_text(template: Text, values: Dict[Text, Text]) -> Text:
     """
 
     try:
-        text = re.sub(r"{([^\n{}]+?)}", r"{0[\1]}", template)
+        text = re.sub(r"{([^\n{}:]+?)}", r"{0[\1]}", template)
```

Comment:
can you please add a test for this?

Reply:
Added several tests which revealed issues in the original fix. Should be ready now.